client/x/evmstaking/keeper/abci.go

```diff
@@ -110,6 +111,16 @@ func (k *Keeper) EndBlock(ctx context.Context) (abci.ValidatorUpdates, error) {
 			return nil, errors.Wrap(err, "burn coins")
 		}
 
+		log.Debug(ctx, "Adding undelegation to withdrawal queue",
+			"delegator", entry.delegatorAddress,
+			"validator", entry.validatorAddress,
+			"max_amount", maxAmount.String())
+		if entry.amount.LT(maxAmount) {
+			log.Debug(ctx, "Undelegation amount is less than max amount",
```

Comment:
This log can be a warning then.
And log the full information of the delegator and validator here

Reply:
58fc5ed